src/InvocableInterface.php

```diff
@@ -10,13 +10,17 @@
 interface InvocableInterface
 {
     /**
-     * This method will be run when the instance is called as a function.
+     * Invokes this instance as a function.
      *
      * For instance:
      * If `$obj` is an instance of the implementing class, doing `$obj()` will invoke this method.
      * That pun was intended.
      *
      * @since [*next-version*]
+     *
+     * @param mixed $argN An argument to the invocation.
```

Comment:
Should be "Any number of arguments", no?

Reply:
That would kinda hint that this is a set or array.

Reply:
Then we need a better way to document variable number of parameters. PSR-5 is missing this information. ApiGen and PHPDocumenter use this format:

```
@param type $name,... The description.
```